pkgs/by-name/an/angryoxide/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  libxkbcommon,
+  sqlite,
+  zlib,
+  stdenv,
+  darwin,
+  wayland,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "angryoxide";
+  version = "0.8.28";
+
+  src = fetchFromGitHub {
+    owner = "Ragnt";
+    repo = "AngryOxide";
+    rev = "v${version}";
+    hash = "sha256-n2c1G8Y9dFV+0MZehCRBAKkzN6XOWfDX0AVWb+o08VI=";
+  };
+
+  cargoHash = "sha256-cir3UNHE7z+pq3oBdBo8U9tcRh6jCCSN/bwuDTg5rrw=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs =
+    [
+      libxkbcommon
+      sqlite
+      zlib
+    ]
+    ++ lib.optionals stdenv.isDarwin [
+      darwin.apple_sdk.frameworks.AppKit
+    ]
```

Comment:
Is this still needed with the new apple-sdk pattern?

Reply:
not needed for sure